streaming/base/storage/util.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for storage."""
+
+import os
+import time
+from typing import Any, Dict, Optional
+
+import boto3
+import requests
+from boto3.s3.transfer import TransferConfig
+from botocore import UNSIGNED
+from botocore.config import Config
+
+
+def download_file_from_s3(local: str,
```

Comment:
Can we please keep this method in the same `download.py` file since all the other cloud provider download functionality is in the same `download.py` file !!!

Reply:
Done